### PR TITLE
[PBW-7596] prevented setting ads volume to 1 if it is already 1

### PR DESCRIPTION
**Pull request by zelibobla**
Setting the value of ads to 1 when it was already 1 was causing an error from Google IMA SDK. Fixed this.
Also added gulp webserver task for debugging purpose.

### CODE REVIEW

File: gulpfile.js [hunk 1]
```diff
@@ -85,8 +86,19 @@ gulp.task('test', shell.task(['make test']));
 
 // Initiate a watch
 gulp.task('watch', function() {
-  gulp.watch(path.scripts, ['browserify']);
+  gulp.watch("js/**/*", ['build']);
 });
 
 // The default task (called when you run `gulp` from cli)
-gulp.task('default', ['build']);
+gulp.task('default', ['build', 'webserver', 'watch']);
```
Comment (aeng7) [inline gulpfile.js]: Was it intentional for these gulpfile changes to make it in this PR?

File: js/google_ima.js [hunk 2]
```diff
@@ -1799,6 +1801,7 @@ require("../html5-common/js/utils/utils.js");
 
               if (this.savedVolume >= 0)
               {
+
```
Comment (aeng7) [inline js/google_ima.js]: Extra line

**Comment by aeng7** [comment]
LGTM other than some comments

File: gulpfile.js (hunk 1 above)
Comment (zelibobla) [reply gulpfile.js]: Nope. Sorry. Fixed

File: js/google_ima.js (hunk 2 above)
Comment (zelibobla) [reply js/google_ima.js]: Fixed

**Comment by aeng7** [comment]
LGTM

**Pull request merged**